### DISCUSSION TITLE
Carattere e dimensioni presente nei documenti

### DISCUSSION THREAD

**Discussion by giadaschiavone** [iisgiua/giuaschool#547] [Richieste di aiuto] [answered]
Buongiorno a tutti. Avrei una domanda.
E' possibile modificare il carrattere presente nei documenti e anche le dimensioni? Per esempio, potrei mettere il carattere Georgia e le dimensioni a 11. In questo caso, come posso fare?

**Comment by trinko** [comment, marked as answer]
Si può fare ma con qualche limite.
Devi personalizzare direttamente i template, che trovi nelle cartelle **templates/pdf** e per i documenti degli scrutini in **templates/coordinatore/documenti**.

Le impostazioni seguono la sintassi dei fogli di stile CSS. Ad esempio, prendiamo il PDF generato per la comunicazione delle credenziali degli alunni: **templates/pdf/credenziali_alunni.html.twig**
Nella prima riga, trovi:
```
<div style="text-align:center;font-family:times;font-size:11pt">
```
Le indicazioni specificano lo stile per il testo che sarà centrato, con font _times_ a 11punti.
Volendo mettere Courier a 14 punti, dovresti modificare così:
```
<div style="text-align:center;font-family:courier;font-size:14pt">
```
 Tieni presente che è supportato solo un limitato set di font. E' possibile estenderlo con altri font, ma le modifiche sono un po' complicate.
 La lista dei font disponibili è la seguente (non li ho testati tutti, ma dovrebbero funzionare):
- aefurat: Furat
- courier : Courier
- dejavusans: DejaVuSans
- dejavusanscondensed: DejaVuSansCondensed
- dejavusansextralight: DejaVuSans-ExtraLight
- dejavusansmono: DejaVuSansMono
- dejavuserif: DejaVuSerif
- dejavuserifcondensed: DejaVuSerifCondensed
- freemono: FreeMono
- freesans: FreeSans
- freeserif: FreeSerif
- helvetica : Helvetica
- symbol : Symbol (solo simboli)
- times : Times New Roman
- zapfdingbats : Zapf Dingbats (solo simboli) 

Per semplicità, non ho riportato i font arabi e quelli delle lingue orientali.

**Reply by giadaschiavone** [reply]
Grazie mille. Comunque, è possibile Calibri oppure no?

**Reply by trinko** [reply]
No, il font Calibri non è tra quelli installati.

**Reply by giadaschiavone** [reply]
D'accordo. Va bene
